Cargo.toml

```diff
@@ -49,13 +49,14 @@ name = "assets"
 path = "examples/06_assets/main.rs"
 
 [dev-dependencies]
-obj-rs = "0.4.16"
 gfx = "0.12"
 
 [dependencies]
-cgmath="0.11"
-glutin = "0.6"
+cgmath = "0.11"
+genmesh = "0.4"
 gfx = "0.12"
 gfx_device_gl = "0.11"
 gfx_window_glutin = "0.12"
-genmesh="0.4"
+glutin = "0.6"
+imagefmt = "*"
+wavefront_obj = "*"
```

Comment:
These shouldn't be wildcarded, could break the example in the future if the dependent crates publish a major change.